src/deepsparse/transformers/helpers.py

```diff
@@ -193,3 +199,93 @@ def _normalize_fields(_dict):
         return result
 
     return _wrapper
+
+
+def truncate_transformer_onnx_model(
```

Comment:
need to add unit tests with multiple zoo models to check that the expected layer is cut to